### DISCUSSION TITLE
Need a pagination in grafana bar chart

### DISCUSSION THREAD

**Discussion by lalitdhami007** [grafana/grafana#55982] [Observability]
Right now there is no pagination in grafan bar chart so it's difficult to show lots of data in bar chart

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
